### ISSUE TITLE
Call for maintainers

### ISSUE THREAD

**Issue by zimme** [prettier/prettier-eslint#275]
I'm thankful for all the understanding people on here and I am sorry that I haven't been able to prioritize prettier-eslint. I've been able to work on prettier-eslint for an hour here and there but it hasn't been enough time to get a good working version out with all the most critical bug fixes to upgrade of eslint.

I don't expect being able to prioritize prettier-eslint any higher in the near future as I've got another baby on the way.

If there are any people that's been active with contributing PR's and discussions on issues I'd like to ask you to step up and help out maintain this project.

I'd also like to ask @prettier/eslint to help out either with maintaining prettier-eslint or help onboard any new maintainers.

I'll do my very best to be available to help new maintainers onboard.

Just to be clear, I'm not stepping down as a maintainer I'm just not able to prioritize this project as it deserves.

Pinging some people that's created PR's lately.
@hamzahamidi @silenaker @kylemh @jondkinney

**Comment by BPScott** [comment]
*waves from @prettier/eslint*

Heya @zimme, I've stayed away from maintaining this repo as I don't use it personally (unlike eslint-plugin-prettier). I can probably help out with small tidyings here and there and some issue triage, but I'm generally unfamiliar with prettier-eslint so wouldn't feel comfortable taking over as a leading maintainer.

That said if you want to give me npm access to publish I'll try and kick the tires on a few things

**Comment by hamzahamidi** [comment]
I'm in for maintaining the github repo. As for the npm repository, I think it's better to add a small CI/CD script to automatize the publishing.

**Comment by zimme** [comment]
Currently, there is CI/CD setup for this project on Travis using semantic release. It will publish a new npm version on any commit that follows the semantic release format. https://github.com/semantic-release/semantic-release#how-does-it-work.

**Comment by hamzahamidi** [comment]
> Currently, there is CI/CD setup for this project on Travis using semantic release. It will publish a new npm version on any commit that follows the semantic release format. https://github.com/semantic-release/semantic-release#how-does-it-work.

I see that the script is triggered whenever a new commit is added to master branch. Why not add reviewers policies to validate a PR (for example a minimum of 2 reviewers from the maintainers is needed to approve a PR)

**Comment by kylemh** [comment]
I'm willing and able.

**Comment by djbeadle** [comment]
@zimme Have you considered adding this project to [JazzBand](https://jazzband.co/)?

> The purpose of the Jazzband is to help Open Source projects on GitHub share responsibility for software maintenance. It’s essentially a GitHub organization that is open to everyone who is willing and able to maintain its projects. It was born out of necessity of expanding the group of maintainers of some projects whose original authors weren’t able to effectively manage them when people volunteered to help out.

**Comment by kylemh** [comment]
It looks like JazzBand is for Python projects.

**Comment by djbeadle** [comment]
😬 Whoops! Missed that part. Maybe there's demand for a similar organization for JS projects?

**Comment by zimme** [comment]
Hey @prettier/eslint could you add @kylemh, @anikethsaha and @hamzahamidi as they have showed an interest in helping out maintaining this project. I don't have any github admin access to add them to the team.

**Comment by BPScott** [comment]
Alas I've not got rights to wrangle that, I think @lipis may though ^^

**Comment by sladyn98** [comment]
@zimme  I'm in for maintaining the github repo.Do let me know if you need any help :)

**Comment by lipis** [comment]
@zimme I made you an admin!

**Comment by hexagon6** [comment]
I would like to contribute as well, as I use this dependency in quite a few projects and have been waiting on updates for quite a while now. :waffle:

**Comment by zimme** [comment]
@hamzahamidi @kylemh I've made you maintainers of this repo and will do the same for `pretties-eslint-cli` if @lipis could make me and admin on that repo too.

@hexagon6 @anikethsaha @sladyn98 I'll consider adding you as maintainers after you've created or helped out with a PR or two just as @hamzahamidi and @kylemh has already done.

**Comment by zimme** [comment]
What I would focus on for now is to get this package ready to support prettier 2 and eslint 7.

Just remember to use the semantic release commit messages as outlined here https://github.com/prettier/prettier-eslint/issues/275#issuecomment-571560038 and new releases will be published as commits are pushed/merged to master.

What I've done sometime when I know I have multiple PRs to merge and changes to commit is to work locally and merge/commit there and once I'm done I've just pushed to master.

**Comment by kylemh** [comment]
I understand that prettier-eslint-cli should be kept updated, but I wonder if this repo should be converted into simply providing an ESLint preset with the prettier plugin pre-configured.

When I used this package it was because there was no interop. I haven’t needed this package for a long time because I can just use the prettier plugin to do formatting and linting at the same time.

**Comment by zimme** [comment]
@kylemh `prettier-eslint-cli` is just a package that uses this package. There is no formatting code within that package, just code to find and format files using this package. So it wouldn't really make sense to keep that one up to date and make this package basically a wrapper around `eslint-plugin-prettier`.

This package is a bit different than `eslint-plugin-prettier` in that this package first runs `prettier` on the code then `eslint --fix` as well as changing eslint rules on the fly based on your prettier config.

Something I often run into when only using `eslint-plugin-prettier`, which also formats the code with `prettier` obviously, is that some `eslint` rules fixes the code in a way that `prettier` would format differently resulting in me having to run `eslint --fix` multiple times to get correct formatting.
This might happen with this package too in some cases but I have yet to run into that scenario from what I can remember.

I do most time, use `eslint-config-prettier` and sometimes even `eslint-plugin-prettier` together with this package, but using `eslint-plugin-prettier` together with this package is mostly to get linting warnings in the editor and not really for any "format on save" functionality that would run `prettier` via `eslint --fix` as an `eslint` rule.

**Comment by anikethsaha** [comment]
What about bringing `prettier-eslint-cli` to this repo as monorepo ?

**Comment by zimme** [comment]
@anikethsaha Would that bring any real value to the project? If it would then I'm ok with it.

**Comment by anikethsaha** [comment]
one can be it will help as we can use the latest `prettier-eslint` API which is not published in `prettier-eslint-cli`.

**Comment by zimme** [comment]
Fair point, however that can be achieved via npm link too if I remember correctly.

But as I said, if value is brought into the project and it's something that the majority of the maintainers of this project wants, I'm good with it.

**Comment by anikethsaha** [comment]
cool 👍

**Comment by kylemh** [comment]
I apologize ahead of time, as I don't use this project anymore, so I probably won't get to doing much until I'm much less busy at work 🙏 

but I have like no life, so I'm sure I'll get around to stuff actively.

**Comment by lipis** [comment]
Did we find any maintainer?

**Comment by zimme** [comment]
@lipis yeah, I've added @kylemh and @hamzahamidi as maintainers of this repo but I need admin access to the prettier-eslint-cli too so I can add them there too

**Comment by lipis** [comment]
@zimme Done!

**Comment by zimme** [comment]
@idahogurl You're now a maintainer too
